js/client-teambuilder.js

```diff
@@ -2570,6 +2584,13 @@
 			}
 			this.chartSet(val, selectNext);
 		},
+		searchChange: function (e) {
+			//91 for right CMD / 93 for left / CMD 17 for CTL
+			if (e.keyCode != 91 && e.keyCode != 93 && e.keyCode != 17) {
```

Comment:
Can use `!==` here as (iirc) the type will always be `number`

Reply:
Thanks for the feedback! I'll update the PR when I'm home